validator/client/validator.go

```diff
@@ -135,53 +142,74 @@ func (v *validator) SlotDeadline(slot uint64) time.Time {
 // beginning of a new epoch.
 func (v *validator) UpdateAssignments(ctx context.Context, slot uint64) error {
 	// Testing run time for fetching every slot. This is not meant for production!
-	if slot%params.BeaconConfig().SlotsPerEpoch != 0 && v.assignment != nil && false {
+	if slot%params.BeaconConfig().SlotsPerEpoch != 0 && v.assignments != nil {
```

Comment:
Do not remove the `false` here, we want to request assignments every slot in case of reorgs for now @shayzluf

Reply:
should i comment TestUpdateAssignments_DoesNothingWhenNotEpochStartAndAlreadyExistingAssignments
as it fails

Reply:
commented it